ansible/roles/slapd/defaults/main.yml

```diff
@@ -151,32 +155,26 @@ slapd__combined_schemas: '{{ slapd__default_schemas
 # .. envvar:: slapd__base_packages [[[
 #
 # List of required APT packages for OpenLDAP service.
-slapd__base_packages: [ 'slapd', 'ldap-utils', 'ssl-cert', 'libldap-common' ]
-
-                                                                   # ]]]
-# .. envvar:: slapd__rfc2307bis_packages [[[
-#
-# List of APT packages to install in preparation to use ``rfc2307bis`` schema
-# instead of the ``nis`` schema.
-slapd__rfc2307bis_packages: [ 'fusiondirectory-schema' ]
+slapd__base_packages:
+  - 'slapd'
+  - 'ldap-utils'
+  - 'ssl-cert'
+  - 'libldap-common'
+  - 'schema2ldif'
 
                                                                    # ]]]
 # .. envvar:: slapd__schema_packages [[[
 #
-# List of APT packages that contain LDAP schemas loaded into the directory by
-# the server. Debian has multiple ``fusiondirectory-*-schema`` and
-# ``gosa-*-schema`` packages that conflict with each other, therefore the list
-# of packages should be synchronized.
-slapd__schema_packages:
-
-  # Support for 'sudo' rules in LDAP
-  - 'fusiondirectory-plugin-sudo-schema'
+# List of APT packages that contain LDAP schemas to be loaded into the
+# directory by the server.
+slapd__schema_packages: []
 
                                                                    # ]]]
 # .. envvar:: slapd__packages [[[
 #
 # List of additional APT packages to install with OpenLDAP service.
 slapd__packages: []
+
```

Comment:
It's just a aesthetic nitpick - last variable in a section shouldn't have an empty line after it and before the fold markers. That way all variables have consistent spacing when you ignore fold markers.

Reply:
Should be fixed in the rebase...I'm not sure I like the style though....not having an empty line after each variable increases the risk that the fold markers end up as part of the value of the variable....